dnf-behave-tests/fixtures/specs/dnf-transaction/wilee-1.spec

```diff
@@ -0,0 +1,29 @@
+Name: wilee
```

Comment:
Alright, one more note, can you actually name this package e.g. `dnf-plugin-remove-llama`, as it is not a random package, we can actually give it a meaningful description.